test/telemetry_influx_db_test.exs

```diff
@@ -204,6 +204,25 @@ defmodule TelemetryInfluxDBTest do
         stop_reporter(pid)
       end
 
+      @tag protocol: protocol
+      test "events are reported with metadata tags specified for #{protocol} API", %{
+        protocol: protocol
+      } do
+        ## given
+        event = given_event_spec([:database, :repo], [:hostname])
+        pid = start_reporter(protocol, %{events: [event]})
+
+        ## when
+        :telemetry.execute([:database, :repo], %{"query_time" => 0.01}, %{hostname: "host-01"})
+
+        ## then
+        assert_reported("database.repo", %{"query_time" => 0.01, "hostname" => "\"host-01\""})
```

Comment:
the second argument of `assert_reported` are values, the third one are tags.
See the tests for tags:
https://github.com/ludwikbukowski/telemetry_influxdb/blob/master/test/telemetry_influx_db_test.exs#L176

However, I just realised that influxdb returns the measurements and tags in sorted order.
There might be a need to fix `assert_reported` function to match the ordering.

Reply:
Yep, I tried yesterday passing the tag as the third argument, but test failed due to InfluxDB returning elements in different order. I didn't want to modify `assert_reported`, so I passed everything as the second parameter.

Reply:
Ok, this fine! I will refactor `assert_reported` in some separate PR